pkg/apis/kosmos/v1alpha1/podconvertpolicy_types.go

```diff
@@ -0,0 +1,132 @@
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:shortName=pc;pcs
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type PodConvertPolicy struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec is the specification for the behaviour of the podConversion.
+	// +optional
+	Spec PodConvertPolicySpec `json:"spec,omitempty"`
+}
+
+type PodConvertPolicySpec struct {
+	// A label query over a set of resources.
+	// If name is not empty, labelSelector will be ignored.
+	// +required
+	LabelSelector *metav1.LabelSelector `json:"labelSelector"`
+
```

Comment:
We should refer to this: [here](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required)